debian/rules

```diff
@@ -51,8 +57,18 @@ override_dh_auto_install:
 
 override_dh_auto_build:
 
+	if ! pkg-config --modversion xfont2 1>/dev/null 2>/dev/null; then \
+	    cp nx-X11/config/cf/host.def nx-X11/config/cf/host.def.bak; \
+	    sed -e 's/#define.*HasXfont2.*YES/#define HasXfont2 NO/' -i nx-X11/config/cf/host.def; \
```

Comment:
Agreeing here. See the latest approach just pushed.

 * FONT_DEFINES and XLIBFONT now get defined in Server.tmpl.
 * both get overridden in the main Makefile (there is the pkg-config based auto-detection code now)
 * debian/control simply build-depends on libxfont-dev (which is API 2 or 1 depending on what Debian version we build against)
 * nx-libs.spec has conditional BRs depending on what distro and distro version we build against